### ISSUE TITLE
The app does not start

### ISSUE THREAD

**Issue by Vitor238** [flathub/com.todoist.Todoist#17]
When clicking to open the app, the app does not start. I also tried running the command below:

`flatpak run com.todoist.Todoist `

and got the following error message:

[3 preload-host-spawn-strategy] Running: /app/bin/zypak-helper child - /app/extra/todoist/todoist --type=zygote
Gtk-Message: 18:06:26.634: Failed to load module "canberra-gtk-module"
Gtk-Message: 18:06:26.635: Failed to load module "canberra-gtk-module"

After restarting the session, the application started working again.

OS: Ubuntu 21.10 - Flatpak 1.10.2

**Comment by jurf** [comment]
Yeah, I reproduced this too. It might be due to the update to 21.08, but I’ll leave it for now, as it works otherwise.

**Comment by raimue** [comment]
I think this works just fine for recent versions. I would like to close this old report. Please comment or reopen if this issue is still relevant.
